### PR TITLE
feat(llmobs): improve langchain llmobs span names

### PR DESCRIPTION
**Pull request by lievan**
Change the span name for langchain auto-instrumented spans to be module + class

APM span name doesn't change -- just the name on the llmobs span event we send to evp

## Checklist

- [x] Change(s) are motivated and described in the PR description
- [x] Testing strategy is described if automated tests are not included in the PR
- [x] Risks are described (performance impact, potential for breakage, maintainability)
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed or label `changelog/no-changelog` is set
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/))
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))
- [x] If this PR changes the public interface, I've notified `@DataDog/apm-tees`.

## Reviewer Checklist

- [x] Title is accurate
- [x] All changes are related to the pull request's stated goal
- [x] Description motivates each change
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes
- [x] Testing strategy adequately addresses listed risks
- [x] Change is maintainable (easy to change, telemetry, documentation)
- [x] Release note makes sense to a user of the library
- [x] Author has acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

File: ddtrace/llmobs/_utils.py [hunk 1]
```diff
@@ -27,6 +28,12 @@ def _get_llmobs_parent_id(span: Span) -> Optional[int]:
     return None
 
 
+def _get_span_name(span: Span) -> str:
+    if span.name == LANGCHAIN_APM_SPAN_NAME and span.resource != "":
+        return span.resource.lower()
```
Comment (Yun-Kim) [inline ddtrace/llmobs/_utils.py:33]: Why use lowercase resource name?

**Review by Yun-Kim — COMMENTED** [review]
This is a good workaround for now, let's write a ticket to solve this in a cleaner way in the future. Just one comment

File: ddtrace/llmobs/_utils.py (hunk 1 above)
Comment (lievan) [reply ddtrace/llmobs/_utils.py:33]: I thought it looked cleaner, also consistent with all lowercase for our other integration span names (intentional/or per-chance) though I don't have strong feelings here

Comment (Yun-Kim) [reply ddtrace/llmobs/_utils.py:33]: Let's keep it the original resource name as formatting the class/module names might cause confusion. Thanks!

**Review by Yun-Kim — APPROVED** [review]

**Pull request merged**